doc/user/content/sql/create-cluster.md

```diff
@@ -35,163 +35,106 @@ you may not create a cluster with a source and an index.
 We plan to remove this restriction in a future version of Materialize.
 {{< /warning >}}
 
-Importantly, clusters are strictly a logical component; they rely on [cluster
-replicas](/get-started/key-concepts#cluster-replicas) to run dataflows. Said a
-slightly different way, a cluster with no replicas does no computation. For
-example, if you create an index on a cluster with no replicas, you cannot select
-from that index because there is no physical representation of the index to read
-from.
-
-Though clusters only represent the logic of which objects you want to bundle
-together, this impacts the performance characteristics once you provision
-cluster replicas. Each object in a cluster gets instantiated on every replica,
-meaning that on a given physical replica, objects in the cluster are in
-contention for the same physical resources. To achieve the performance you need,
-this might require setting up more than one cluster.
-
-{{< warning >}}
-Clusters containing sources and sinks can have at most one replica.
-
-We plan to remove this restriction in a future version of Materialize.
-{{< /warning >}}
```

Comment:
This warning should stay but maybe change to say "have a replication factor of at most one"?